control/cli.py

```diff
@@ -1368,11 +1386,69 @@ def format_size(self, sz):
                 sz = int(sz)
                 return f"{sz} GiB"
             return f"{sz:2.1f} GiB"
-        sz = sz / (1024.0 * 1024.0 * 1024.0 * 1024.0)
-        if sz == int(sz):
-            sz = int(sz)
-            return f"{sz} TiB"
-        return f"{sz:2.1f} TiB"
+        if sz < 1024 * 1024 * 1024 * 1024 * 1024:
+            sz = sz / (1024.0 * 1024.0 * 1024.0 * 1024.0)
+            if sz == int(sz):
+                sz = int(sz)
+                return f"{sz} TiB"
+        return f"{sz:2.1f} PiB"
+
+    def get_size_in_bytes(self, sz):
```

Comment:
Generally B vs iB

```
Unit of Measure   Bytes

Kilobyte (kB)	  1000¹ = 10³  = 1,000
Megabyte (MB)	  1000² = 10⁶  = 1,000,000
Gigabyte (GB)	  1000³ = 10⁹  = 1,000,000,000
Terabyte (TB)	  1000⁴ = 10¹² = 1,000,000,000,000
Petabyte (PB)	  1000⁵ = 10¹⁵ = 1,000,000,000,000,000

Kibibyte (KiB)    1024¹ = 2¹⁰  = 1,024
Mebibyte (MiB)    1024² = 2²⁰  = 1,048,576
Gibibyte (GiB)    1024³ = 2³⁰  = 1,073,741,824
Tebibyte (TiB)    1024⁴ = 2⁴⁰  = 1,099,511,627,776
Pebibyte (PiB)    1024⁵ = 2⁵⁰  = 1,125,899,906,842,624


Additional Info:

1 GB  = 1000 MB  = 1,000,000 kB  = 1,000,000,000 Bytes

1 GiB = 1024 MiB = 1,048,576 KiB = 1,073,741,824 Bytes
```

Reply:
@baum I know, but nobody really makes this distinction. So, I just used the binary values. Not to confuse the users.

Reply:
@gbregman actually the implementation that is distinguishing MB and MiB but saying those are the same number of bytes is confusing 🤷🏼‍♂️

Reply:
Changed that.